app/services/evaluator_management_service.rb

```diff
@@ -22,6 +22,7 @@ def remove_evaluator(evaluator_type, evaluator_id)
   def self.accept_evaluator_invitation(user)
     invitations = EvaluatorInvitation.where(email: user.email)
     invitations.each do |invite|
+      user.update(first_name: invite.first_name, last_name: invite.last_name)
```

Comment:
fixes a bug in the invite flow where it doesn't keep the name of the invited user